### PR TITLE
Orgrim now has appropriate claims & Bronzebeard brothers are no longer half brothers

### LINKED ISSUE #509: Orgrim Claim
**Describe your suggestion in full detail below:**
Orgrim should have a claim on the Horde so he can challenge the Warchief.
<details>
<summary>Click to expand</summary>

image

</details>

### PR DESCRIPTION
**Pull request by elcascavel**
## Changelog:
- Gave Orgrim a pressed claim to the Horde title and the kingdom level title of Blackrock;
- Orgrim can now challenge the Warchief to a Mak'gora;
- Magni, Muradin and Brann are now listed as brothers. Couldn't find any information on their mother so I added a random one with a dwarven female name;
- Kurdran and Falstad are no longer brothers;
- Replaced all `add_claim` references with `add_pressed_claim`.

Fixed #509 and #405

### CODE REVIEW

File: history/characters/17000_28000_bronzebeard.txt [hunk 1]
```diff
@@ -368,10 +371,20 @@
...
 	511.7.14={ death=yes }
 }
 
+17043={
+	name = Kusahe
+	female = yes
+	culture=bronzebeard religion=khazism
+	trait=education_diplomacy_3
+	add_spouse = 17042
+	275.2.12={ birth=yes trait=creature_dwarf immortal_age=18 }
```
Comment (zumbak04) [inline history/characters/17000_28000_bronzebeard.txt:385]: This is CK2 leftover.
```suggestion
	275.2.12={ birth=yes trait=creature_dwarf }
```

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: history/characters/10000_orc.txt [hunk 2]
```diff
@@ -610,8 +610,8 @@
 		employer = 10000
 		add_prestige = 750		# Should be able to invite clans when he's Warchief
 		add_piety = 375
-		add_claim = e_horde
-		add_claim = k_orc_blackrock
+		add_pressed_claim = e_horde
```
Comment (zumbak04) [inline history/characters/10000_orc.txt:613]: Could you replace other `add_claim` with `add_pressed_claim` to fix all kinds of this bug?

Comment (elcascavel) [reply history/characters/10000_orc.txt:613]: As requested, I replaced all references with `add_pressed_claim`: 0da8ab8dd2c90f253fe531bf6a5862ee03542889

**Review by zumbak04 — APPROVED** [review]
LGTM, good job!

**Pull request merged**